ExRam.Gremlinq.Core/Models/IGraphModel.cs

```diff
@@ -9,7 +9,10 @@ public interface IGraphModel
 
         IGraphElementModel VerticesModel { get; }
         IGraphElementModel EdgesModel { get; }
+        IMetadataStore MetadataStore { get; }
```

Comment:
I think this is cool for now, I don't know whether I want this on the GraphModel. Maybe the methods from IMetadataStore should rather go on IGraphModel right away, but it's good for now.

Reply:
I went ahead and removed the MetadataStore concept and wrapped it up in the ElementBuilder, since it should only be manipulated during the 'Configure' call.